### PR TITLE
Add storage only benchmark.

### PR DESCRIPTION
**Pull request by grao1991**
### Description
Add a skeleton of storage only benchmark.

### CODE REVIEW

**Review by zjma — APPROVED** [review]

**Review by msmouse — APPROVED** [review]
beautiful

File: execution/executor-benchmark/src/fake_executor.rs [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+type Address = [u8; 32];
+
+static TOTAL_SUPPLY_STATE_KEY: Lazy<StateKey> = Lazy::new(|| StateKey::TableItem {
+    handle: "1b854694ae746cdbd8d44186ca4929b2b337df21d1c74633be19b2710552fdca"
+        .parse()
+        .unwrap(),
+    key: vec![
+        6, 25, 220, 41, 160, 170, 200, 250, 20, 103, 20, 5, 142, 141, 214, 210, 208, 243, 189, 245,
+        246, 51, 25, 7, 191, 145, 243, 172, 216, 30, 105, 53,
+    ],
```
[the remark this answers is not in the document]
Comment (grao1991) [reply execution/executor-benchmark/src/fake_executor.rs:45]: Done

File: execution/executor-benchmark/src/fake_executor.rs [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+
+        let mut new_account: Account = Default::default();
+        new_account.authentication_key = new_account_address.to_vec();
+
+        let mut new_coin_store: CoinStore = Default::default();
+        new_coin_store.coin = account_creation_info.initial_balance;
+
+        let mut total_supply: u128 =
+            Self::get_value(&TOTAL_SUPPLY_STATE_KEY, &state_view)?.unwrap();
+        total_supply -= gas as u128;
```
[the remark this answers is not in the document]
Comment (grao1991) [reply execution/executor-benchmark/src/fake_executor.rs:230]: Done

File: execution/executor/src/block_executor.rs [hunk 3]
```diff
@@ -109,22 +129,25 @@ where
     }
 }
 
-struct BlockExecutorInner<V> {
+struct BlockExecutorInner<V, T> {
     db: DbReaderWriter,
     block_tree: BlockTree,
-    phantom: PhantomData<V>,
+    phantom0: PhantomData<V>,
+    phantom1: PhantomData<T>,
```
Comment (lightmark) [inline execution/executor/src/block_executor.rs:136]:
```suggestion
    phantom: PhantomData<(V, T)>,
```

Comment (grao1991) [reply execution/executor/src/block_executor.rs:136]: Done

**Pull request merged**